go.work

```diff
@@ -0,0 +1,13 @@
+go 1.18
+
+use (
+	./op-batcher
+	./op-e2e
+	./op-node
+	./op-proposer
+)
+
+replace github.com/ethereum/go-ethereum v1.10.16 => github.com/ethereum-optimism/reference-optimistic-geth v0.0.0-20220512193241-32ef0bba48ce
```

Comment:
I notice that this is replaced on `op-e2e/go.mod` as well - I'm assuming that the replace there will override this replace, I feel like it would make sense to have a single replace that applies to the entire repo to ensure that things are all on the same version of `go-ethereum`

Reply:
The replacement does two things: set the version, and deal with the fact that our reference geth declares itself to be a different module from how it is imported as. As such, I think you might need the replace, but I'm not fully sure.

Reply:
We need both replacements. The one in `go.work` sets the replacement globally for dev purposes. The one in `op-e2e` lets us import the real reference geth package in CI.